libopenage/error/stackanalyzer.cpp

```diff
@@ -245,7 +291,8 @@ void StackAnalyzer::get_symbols(std::function<void (const backtrace_symbol *)> c
 	symbol.lineno = 0;
 
 	if (reversed) {
-		for (size_t idx = this->stack_addrs.size(); idx > 0; idx--) {
+		// `for (auto pc : this->stack_addrs | <ranges-ns>::view::reverse)` after Ranges-TS
+		for (size_t idx = this->stack_addrs.size(); idx-- > 0;) {
```

Comment:
The next line does a dereference using `idx`. I was initially planning to use a reverse iterator, but then got a hint from line 181 above to make a smaller change.